app/controllers/ApplicationController.java

```diff
@@ -23,16 +19,12 @@
 public final class ApplicationController extends SunriseController {
 
     @Inject
-    public ApplicationController(final PlayJavaSphereClient client, final CategoryTree categoryTree, final Configuration configuration) {
-        super(client, categoryTree, configuration);
+    public ApplicationController(final PlayJavaSphereClient client, final CategoryTree categoryTree,
+                                 final Configuration configuration, final TemplateService templateService, final CmsService cmsService) {
+        super(client, categoryTree, configuration, templateService, cmsService);
     }
 
     public F.Promise<Result> index() {
```

Comment:
But then I have to apply `Promise.pure` in the `HomeController` instead (that's why I put the `pure` in `ApplicationController`). It's just a temporary method that will eventually change into a `Promise.pure`, that's why I chose to put the pure in here.